packages/react-core/src/components/Text/__tests__/TextList.test.tsx

```diff
@@ -41,6 +41,11 @@ test('Renders as "dl" element when component="dl"', () => {
   expect(screen.getByText('Test')).toHaveProperty('nodeName', 'DL');
 });
 
+test('Renders with plain modifier class when isPlain is set to true', () => {
+  render(<TextList isPlain>Test</TextList>);
+  expect(screen.getByText('Test')).toHaveClass('pf-m-plain');
```

Comment:
Can you use `screen.getByRole('list'))` here instead to get the element?

Reply:
Yep! Though I just reused that from the other tests - should I update any of the other ones? https://github.com/patternfly/patternfly-react/blob/ab5dc68a6ee80eb4151636e4a0e92b3dec66a237/packages/react-core/src/components/Text/__tests__/TextList.test.tsx

Reply:
That would be ideal, but I wouldn't consider it a requirement.